From d98792c0c89de92a50a9f1554a83efab86cdd699 Mon Sep 17 00:00:00 2001
From: timyao <tim.yao@amlogic.com>
Date: Fri, 1 Aug 2014 17:44:11 -0700
Subject: [PATCH 4636/5965] PD #94691. Add H.264 video decoder dynamic clock
 adjustment on M8 baby and M8m2. Fix M8m2 when M8 binary compatibility changes
 are merged.

Change-Id: Idd53417b418d01557699d7628191266f6050ffcc
---
 drivers/amlogic/amports/vh264.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/amlogic/amports/vh264.c b/drivers/amlogic/amports/vh264.c
index 46f1f3cd15b7..d000b2b82cf2 100755
--- a/drivers/amlogic/amports/vh264.c
+++ b/drivers/amlogic/amports/vh264.c
@@ -345,6 +345,10 @@ static void vdec_dfs(void)
 {
 #if MESON_CPU_TYPE >= MESON_CPU_TYPE_MESON8B
     vdec_power_mode((fifo_level() > DFS_HIGH_THEASHOLD) ? 0 : 1);
+#elif MESON_CPU_TYPE == MESON_CPU_TYPE_MESON8
+    if (IS_MESON_M8M2_CPU) {
+        vdec_power_mode((fifo_level() > DFS_HIGH_THEASHOLD) ? 0 : 1);
+    }
 #endif
 }
 
-- 
2.19.0

